deploy/control.prod.toml

```diff
@@ -16,7 +27,8 @@ hostname = "gpuctl.perial.co.uk"
 port = 443
 
 [Onboard.Remote.Satellite]
+# intervals in nanoseconds
```

Comment:
Ooof that some of these are ns and some are secs. I should land1ff7eccde66bbf36bf63e012e245ff9685b483a9, so we can write `foo = "1s"`